include/volume/math_helpers.h

```diff
@@ -0,0 +1,44 @@
+// VolEsti (volume computation and sampling library)
```

Comment:
for consistency please use `.hpp` instead of `.h`

Reply:
renamed